packages/scandipwa/src/component/CheckoutDeliveryOption/CheckoutDeliveryOption.style.scss

```diff
@@ -77,6 +84,10 @@
         strong {
             display: inline-block;
         }
+
+        &_isDisabled {
+            color: green;
+        }
```

Comment:
Is this block even needed? If so then for what reason?
+ instead of `color: green` use `color: var(--primary-success-color);`